src/bin/bat/clap_app.rs

```diff
@@ -422,11 +433,17 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      '--style=\"..\"' option to the configuration file or export the \
                      BAT_STYLE environment variable (e.g.: export BAT_STYLE=\"..\").\n\n\
                      Possible values:\n\n  \
-                     * full: enables all available components (default).\n  \
-                     * auto: same as 'full', unless the output is piped.\n  \
+                     * auto: enables all components and 'header', unless the output is piped.
+                     * (default)\n  \
+                     * full: enables all available components.\n  \
                      * plain: disables all available components.\n  \
                      * changes: show Git modification markers.\n  \
-                     * header: show filenames before the content.\n  \
+                     * header: displays the filename and filesize.\n \
+                     * header-full: displays all header-* fields.\n \
```

Comment:
If we go with option 1 above I think we can also postpone adding `header-full` but maybe add it in a follow-up PR.